tests/rest/client/v2_alpha/test_auth.py

```diff
@@ -282,9 +241,11 @@ def test_ui_auth(self):
             },
         )
 
-    def test_cannot_change_body(self):
+    def test_can_change_body(self):
         """
-        The initial requested client dict cannot be modified during the user interactive authentication session.
+        The client dict can be modified during the user interactive authentication session.
+
+        Note that this is not spec compliant, but is necessary for clients to work.
```

Comment:
Might be worth putting in a "The test will be removed in the future" notice here too.

Reply:
It should not be removed in the future, it should be updated to show that this gets rejected. (Essentially backing out these changes.

Reply:
@anoadragon453 I updated this comment, hopefully it is clearer now!